### ISSUE TITLE
Add option to hide 'internal' qubes in qube manager

### ISSUE THREAD

**Issue by marmarta** [QubesOS/qubes-issues#8042]
### The problem you're addressing (if any)

It would be nice to add a Internal filter to qube manager

Potentially even set it to not show internal qubes by default?

### The solution you'd like

Have a View -> Show/hide internal qubes or Filter -> [] Internal

### The value to a user, and who that user might be

Less technical users; people with complex setups.

**Comment by andrewdavidwong** [comment]
Related: #6604

**Comment by cfm** [comment]
Thanks for filing this, @marmarta!  Our use-case is described in freedomofpress/securedrop-workstation#857, where we want to prevent users from accessing SecureDrop Workstation‒provisioned VMs and their applications except through specific entry-points we provide.  For the purpose of this ticket, we also want to prevent users from being able to reconfigure SecureDrop Workstation‒provisioned VMs in the Qube Manager, for example by selecting a kernel other than the grsec-hardened one we provide.

**Comment by DemiMarie** [comment]
@cfm Obviously, this will not prevent someone from doing this _intentionally_ (you need sys-gui for that).

**Comment by andrewdavidwong** [comment]
> Thanks for filing this, @marmarta! Our use-case is described in [freedomofpress/securedrop-workstation#857](https://github.com/freedomofpress/securedrop-workstation/pull/857), where we want to prevent users from accessing SecureDrop Workstation‒provisioned VMs and their applications except through specific entry-points we provide. For the purpose of this ticket, we also want to prevent users from being able to reconfigure SecureDrop Workstation‒provisioned VMs in the Qube Manager, for example by selecting a kernel other than the grsec-hardened one we provide.

This, to me, sounds like a much more general feature request than the one described in this issue. I might be wrong, but I would think that trying to use any future "hide internal qubes in Qube Manager" feature in order to hide *other* types of qubes would likely constitute an unintended and out-of-scope (ab)use of that feature. If so, I'd suggest opening a separate feature request issue.

**Comment by cfm** [comment]
Thanks, @andrewdavidwong.  I'll flag that distinction in freedomofpress/securedrop-workstation#857, and we'll stay tuned for clarification.  :-)

We've drafted a follow-up ticket to request this more-general feature, which we'll file next week once we've confirmed the intended scope on our side.
